Pod/Classes/SEGMixpanelIntegrationFactory.m

```diff
@@ -20,11 +20,25 @@ - (instancetype)init
     return self;
 }
 
+
++ (instancetype)instanceWithLaunchOptions: (NSString *)token launchOptions:(NSDictionary *)launchOptions
+{
+    static dispatch_once_t once;
+    static SEGMixpanelIntegrationFactory *sharedInstance;
+    dispatch_once(&once, ^{
+        [Mixpanel sharedInstanceWithToken:token launchOptions:launchOptions];
```

Comment:
Don't initialize this here. Keep the launchOptions as a field in the factory. Then when `createWithSettings` is called, supply the launchOptions to the MixpanelIntegration.